pkg/catalogv2/helm/release.go

```diff
@@ -79,6 +82,7 @@ func ToRelease(obj runtime.Object, isNamespaced IsNamespaced) (*v1.ReleaseSpec,
 	return nil, ErrNotHelmRelease
 }
 
+// getReleaseDataAndKind receives a runtime.Object as input and returns the release data
```

Comment:
// getReleaseDataAndKind receives a runtime.Object which can be a
// unstructured struct or configmap or secret. It extracts the data["release"]
// based on the object type and returns it.